gusto/spatial_methods/spatial_methods.py

```diff
@@ -15,19 +15,20 @@ class SpatialMethod(object):
     The base object for describing a spatial discretisation of some term.
     """
 
-    def __init__(self, equation, variable, term_label):
+    def __init__(self, equation, variable, term_labels):
```

Comment:
We discussed this offline but putting it here for posterity: could we avoid some of the complicated logic below by using `*term_labels` here, or similar?

Can we still enforce that the original labelled form still only has one valid term to be replaced?